gammapy/spectrum/tests/test_make.py

```diff
@@ -87,3 +92,20 @@ def test_spectrum_dataset_maker_hess_cta(
 
     assert_allclose(datasets[0].background.data.sum(), 2.238345, rtol=1e-5)
     assert_allclose(datasets[1].background.data.sum(), 2.164593, rtol=1e-5)
+
+
+def test_safe_mask_maker(spectrum_dataset_maker_crab, safe_mask_maker, observations_hess_dl3):
+
```

Comment:
Never put empty lines at the start of functions (consistency coding style).

Reply:
Done